src/components/common/animations/SpinnerCheckMark/SpinnerCheckMark.js

```diff
@@ -1,49 +1,88 @@
 import React from 'react'
 import Lottie from 'lottie-react-native'
+import { View } from 'react-native'
 import { cloneDeep } from 'lodash'
-
 import AnimationBase from '../Base'
-
+import { isMobileNative } from '../../../../lib/utils/platform'
 import animationData from './data.json'
 
 class SpinnerCheckMark extends AnimationBase {
+  state = {
+    speed: 1,
+    isFinish: false,
+  }
+
   onMount = () => {
-    this.anim.onEnterFrame = e => {
-      const { success } = this.props
-      if (e.currentTime > 130.5 && !success) {
-        this.anim.goToAndPlay(0, true)
+    if (isMobileNative === false) {
+      this.anim.onEnterFrame = e => {
+        const { success } = this.props
+        if (e.currentTime > 130.5 && !success) {
+          this.anim.goToAndPlay(0, true)
+        }
+      }
+
+      this.anim.onComplete = () => {
+        const { onFinish } = this.props
+        if (typeof onFinish === 'function') {
+          onFinish()
+        }
       }
     }
 
-    this.anim.onComplete = () => {
-      const { onFinish } = this.props
-      if (typeof onFinish === 'function') {
-        onFinish()
+    if (isMobileNative) {
+      this.anim.play(0, 130)
+    } else {
+      this.anim.play()
+    }
+  }
+
+  onFinish = () => {
+    const { onFinish, success } = this.props
+
+    if (isMobileNative) {
+      if (!success) {
+        return this.anim.play(0, 129)
+      } else if (!this.state.isFinish) {
+        this.setState({ isFinish: true })
+        return this.anim.play(130, 230)
       }
     }
 
-    this.anim.play()
+    if (typeof onFinish === 'function') {
+      onFinish()
+    }
   }
 
   componentDidUpdate(prevProps) {
     if (prevProps.success === false && this.props.success === true) {
       //speed up when finished
-      this.anim.setSpeed(1.5)
+      if (isMobileNative) {
+        this.setState({
+          speed: 1.5,
+        })
+      } else {
+        this.anim.setSpeed(1.5)
+      }
     }
   }
 
   render() {
     const { height = 196, width = 196 } = this.props
     return (
-      <Lottie
-        ref={this.setAnim}
-        source={cloneDeep(animationData)}
-        style={{
-          marginTop: -height / 2.4,
-          width,
-          height,
-        }}
-      />
+      <View>
```

Comment:
@MathiasMoralesDixtra Just curious, what's the purpose of this wrapper?

Reply:
For some reason, if it doesn't have a view wrapping the content is not centered.